assets/prometheus-operator/cluster-role.yaml

```diff
@@ -4,29 +4,9 @@ metadata:
   labels:
     app.kubernetes.io/component: controller
     app.kubernetes.io/name: prometheus-operator
-    app.kubernetes.io/version: v0.38.1
+    app.kubernetes.io/version: v0.39.0
   name: prometheus-operator
 rules:
-- apiGroups:
-  - apiextensions.k8s.io
-  resources:
-  - customresourcedefinitions
-  verbs:
-  - create
-- apiGroups:
-  - apiextensions.k8s.io
-  resourceNames:
-  - alertmanagers.monitoring.coreos.com
```

Comment:
If we remove this, what will be managing CRDs?

Reply:
Ah yes thanks! That explains your failure from your PR to bump prom-operator. Forgot it was this release that removes it. https://issues.redhat.com/browse/MON-1084 We have this in the next sprint, will just do it then, nothing urgent from this PR as mentioned in description.

/hold